alpha_automl/automl_api.py

```diff
@@ -20,13 +21,12 @@
 
 class BaseAutoML():
 
-    def __init__(self, output_folder, time_bound=15, metric=None, split_strategy='holdout', time_bound_run=5, task=None,
```

Comment:
Don't remove the param `output_folder`, make it optional, like this: `(self, time_bound=15, output_folder=None, metric=None, split_strategy='holdout', time_bound_run=5, task=None, score_sorting='auto', metric_kwargs=None, split_strategy_kwargs=None, start_mode='auto', verbose=logging.INFO)`
If it's None, use `tempfile.mkdtemp()`. If the user provides a path, use that folder or create it  (if it doesn't exist).

Reply:
Resolved.